### PR TITLE
build(deps): bump versions

### PR DESCRIPTION
**Pull request by driskull**
build(deps): bump versions

Stencil 2.1.0: https://github.com/ionic-team/stencil/blob/master/CHANGELOG.md#-210-2020-10-20

I couldn't bump `autoprefixer` because it looks like it's not working with the latest stencil postCSS plugin?

### CODE REVIEW

File: src/assets/styles/global.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@import "./node_modules/@esri/calcite-colors/colors.scss";
+@import "./node_modules/@esri/calcite-colors/src/colors.scss";
```
Comment (jcfranco) [inline src/assets/styles/global.scss:1]: Unrelated: Maybe we should include the sass file as part of the distributable too and not ship the source.

**Comment by macandcheese** [comment]
👍

**Review by macandcheese — APPROVED** [review]

File: .storybook/utils.tsx [hunk 2]
```diff
@@ -15,28 +15,28 @@ import {
   text
 } from "@storybook/addon-knobs";
 
-import colors from "../node_modules/@esri/calcite-colors/colors.json";
+import { Colors } from "../node_modules/@esri/calcite-colors/dist/colors";
```
Comment (jcfranco) [inline .storybook/utils.tsx:18]: Unrelated: Should this be lowercased? I'd expect a class if uppercased.

**Review by jcfranco — APPROVED** [review]
👍 Thanks for the bump! 

Sorry for the unrelated comments. If they make sense, I can submit PRs for those things.

File: src/assets/styles/global.scss (hunk 1 above)
Comment (driskull) [reply src/assets/styles/global.scss:1]: Yeah I noticed that too. I think they make sense, can we update them after?

File: .storybook/utils.tsx (hunk 2 above)
Comment (driskull) [reply .storybook/utils.tsx:18]: Agreed

**Review by jcfranco — APPROVED** [review]
Awesome! 🎉🎉🎉

**Pull request merged**